index.html

```diff
@@ -1,329 +1,328 @@
+      <section class="aboutMe">
+        <h1>About me</h1>
+
+        <p>
+          Hello I’m a software developer! I can help 
+          you build a product , feature or website 
+          Look through some of my work and 
+          experience! If you like what you see and 
+          have a project you need coded, don’t 
+          hestiate to contact me.
+        </p>
+
+        <div>
+          <button type="submit" class="orange-button orange-button3">Get My Resume</button>
+        </div>
+      </section>
+
+      <hr class="line">
+
+      <section class="categories">
+        <ul class="cat">
+          <li class="language">
+            <h2>Languages</h2>
+
+            <ul class="list">
+              <li class="lang">Javascript</li>
+              <li class="lang">Ruby</li>
+              <li class="lang">Html</li>
+              <li class="lang">Css</li>
+            </ul>
+          </li>
+
+          <li class="frameworks">
+            <h2>Frameworks</h2>
+
+            <ul class="list1 list">
+              <li class="frame">Bootstrap</li>
+              <li class="frame">Ruby on Rails</li>
+              <li class="frame">RSPec</li>
+              <li class="frame">CapyBara</li>
+              <li class="frame">Selenium</li>
+            </ul>
+          </li>
+
+          <li class="skills">
+            <h2>Skills</h2>
+          
+            <ul class="list2 list skills">
+              <li class="skill">CodeKit</li>
+              <li class="skill">GitHub</li>
+              <li class="skill">Codepen</li>
+              <li class="skill">Gitlab</li>
+              <li class="skill">Terminal</li>
+            </ul>
+          </li>
+        </ul>
+      </section>
+    </section>
+  </main>
+
+  <footer>
+    <section class="form">
+      <div class="remarks">
+        <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      </div>
+      
+      <form action="https://formspree.io/f/xjvjregp" method="post">
+        <fieldset>
+          <legend>Get in touch</legend>
+          <ul>
+            <li>
+              <label for="name">Name:</label>
+              <input type="text" id="name" name="full name" placeholder="Full name" maxlength="30" required>
+            </li>
+            
+            <li>
+              <label for="email">Email:</label>
+              <!-- only email addresses are accepted and it is a required field -->
+              <input type="email" id="email" name="email address" placeholder="Email address" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required>
+            </li>
+            
+            <li>
+              <label for="msg">Message:</label>
+              <textarea id="msg" name="text" placeholder="Enter text here" maxlength="500" required></textarea>
+            </li>
+    
+            <li>
+              <button type="submit" class="orange-button orange-button4">Get in touch</button>
```

Comment:
- [ ] **This button is failing the contrast test. Please fix it**

You can use [this website](https://www.aditus.io/button-contrast-checker/) to test the contrast on elements.